apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -121,6 +121,28 @@ export function InstructorCourseAdminInstances({
                 </tbody>
               </table>
             </div>
+            ${courseInstances.length === 0
+              ? html`
+                  <div class="my-4">
+                    <p class="text-center font-weight-bold">No course instances found.</p>
+                    <p class="text-center mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
+                    </p>
+                    <p class="text-center">
+                      To create one, click <code>Add course instance</code>.
```

Comment:
I think we should use something other than `<code>` for referring to buttons, because it uses a monospaced font that's different to the actual buttons. Elsewhere we've used `<strong>` to refer to buttons, but this is just an idea.

Reply:
Hi Matt. I agree, and made this change:

<img width="1710" alt="image" src="https://github.com/user-attachments/assets/6bcf31fd-7b4e-4a1e-8d89-e9757e93263b">